server/queries/addQuestions.js

```diff
@@ -0,0 +1,15 @@
+const addQuestions = async (questionsArray, questionsCollection) => {
+  try {
+    const { acknowledged, insertedIds } = await questionsCollection.insertMany(
+      questionsArray,
+    );
+  } catch (e) {
+    return new Error(e.message);
+  }
+
+  if (acknowledged === true) {
```

Comment:
on this query, call getDb() and get collection 'questions', instead of doing in the handler 👍 remove second argument `questionsCollection`